docs/index.md

```diff
@@ -1,58 +1,44 @@
 ---
-id: index
-title: Welcome to Webaverse
+id: index 
+title: Overview 
 hide_title: true
 ---
 
 <div style='margin-top: -60px'></div>
-<h1 align="center"><b>🌌 Welcome To A Universe Of Apps ⚙️</b></h1>
+<h1 align="center"><b>Welcome To Webaverse</b></h1>
 
-<img alt="Webaverse world demo" src="/img/demo.gif" width="40%">
+<br/>
 
-<div align="center">
-  <a href="https://webaverse.com" target="_blank">Site</a>
-  &mdash;
-  <a href="https://github.com/webaverse/" target="_blank">Github</a>
-  &mdash;
-  <a href="https://discord.gg/MQNUGgB" target="_blank">Discord</a>
-  &mdash;
-  <a href="https://twitter.com/webmixedreality" target="_blank">Twitter</a>
-</div>
+These docs will help you navigate and create in <a href="https://webaverse.com/" target="_blank" rel="noopener noreferrer">Webaverse</a>.
 
-<h3 align="center">⚠️ M3 is hosting a hackathon for Webaverse creations! [Find more information on participating here!](m3-hackathon.md) 🔍</h3>
+The Webaverse ecosystem is built on existing open standards and technologies like WebGL, WebXR, IPFS, and Ethereum.
 
-## ❓ What is the Webaverse?
+## Webaverse 
 
-<a href="https://webaverse.com/" target="_blank" rel="noopener noreferrer">The Webaverse</a> is a blanket term used to describe a set of technologies that enable the use of spatial applications and objects powered by WebXR. These spatial applications are called "XRPackages" (XRPKs).
+In this initial section you can go from zero understanding to knowing exactly what Webaverse is.
 
-See [`What Is Webaverse?`](about.md) for a more in-depth explanation, or read on to get started with using and developing XRPKs!
+You'll learn that you can buy and use [Grease](./webaverse/grease) to mint [NFTs](./webaverse/nfts), just by messaging with a [Discord Bot](./webaverse/discord-bot).
 
-See [`Getting Started`](getting-started.md) for detailed guides on how to get started, as a user or developer!
+Or that you can create your very own [Avatars](./webaverse/avatars) and build on [Parcels](./webaverse/parcels) that are a part of everchanging [Seasons](./webaverse/seasons).
 
-## ❓ What Are XRPackages (XRPKs)?
+## Market 
```

Comment:
I think we should decide whether the audience of this is people looking for the market or people looking to create and prioritize the narrative accordingly.

I think these docs are probably more intended for the create side. But if that's true then we should probably start there, not with the Market, unless the intent is to introduce the Market concept first to teach by example.